src/generated/todo/index.js

```diff
@@ -16,8 +16,8 @@ const actions = Object.assign(
 );
 
 const sagas = [].concat(
```

Comment:
This makes `allSagas.js` a bit messy - thoughts?
@d4ncer @qisaw - you'd need to write them all out again.